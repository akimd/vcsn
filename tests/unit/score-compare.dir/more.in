 1.50s: a.is_proper()        # a = "", 2000000x
 2.20s: b.format("text")     # b = [abc] -> B, 1000000x
