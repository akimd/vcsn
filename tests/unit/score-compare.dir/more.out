[32m[1m 1.20[0m [31m[1m 1.50[0m a.is_proper()        # a = "", 2000000x
[31m[1m 2.70[0m [32m[1m 2.20[0m b.format("text")     # b = [abc] -> B, 1000000x
